### PR TITLE
Make deployment instructions a bit more readable

### PR DESCRIPTION
**Pull request by leshi**

### CODE REVIEW

File: u2f-gae-demo/README.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+  u2f.EXTENSION_ID = 'pfboblefjcgdjicmnffhdgionmgcdmne';
+```
+
+In order to deploy to https://noext-dot-u2fdemo.appspot.com, set the version value to ```noext``` in [appengine-web.xml](https://github.com/google/u2f-ref-code/blob/master/u2f-gae-demo/war/WEB-INF/appengine-web.xml):
+```
+  <version>noext</version>
+```
+
+## CrxJs
+In order to configure the appspot server to call the U2F extension you have, set Set the extension id in [u2f-api.js](https://github.com/google/u2f-ref-code/blob/master/u2f-gae-demo/war/js/u2f-api.js) to ```kmendfapggjehodndflmmgagdbamhnfd```:
```
Comment (juanlang) [inline u2f-gae-demo/README.md]: s/set Set/set/

File: u2f-gae-demo/README.md [hunk 2]
```diff
@@ -1,18 +1,28 @@
...
+```
+  u2f.EXTENSION_ID = 'kmendfapggjehodndflmmgagdbamhnfd';
+```
+ 
+In order to deploy to https://crxjs-dot-u2fdemo.appspot.com, set et the version value to ```crxjs``` in [appengine-web.xml](https://github.com/google/u2f-ref-code/blob/master/u2f-gae-demo/war/WEB-INF/appengine-web.xml):
```
Comment (juanlang) [inline u2f-gae-demo/README.md]: s/set et/set/

File: u2f-gae-demo/README.md (hunk 1 above)
Comment (leshi) [inline u2f-gae-demo/README.md]: done

File: u2f-gae-demo/README.md (hunk 2 above)
Comment (leshi) [inline u2f-gae-demo/README.md]: done

**Comment by juanlang** [comment]
lgtm

**Pull request merged**